subsys/bluetooth/services/fast_pair/fp_advertising.c

```diff
@@ -30,8 +30,6 @@ static const uint16_t fast_pair_uuid = BT_FAST_PAIR_SERVICE_UUID;
 static const uint8_t flags;
 static const uint8_t empty_account_key_list;
 
-static const enum fp_field_type ak_filter_type = FP_FIELD_TYPE_SHOW_UI_INDICATION;
```

Comment:
I would leave it as it is (just remove `const`) and update using a dedicated function. See comments above.